installation_tips/README.md

```diff
@@ -1,36 +1,53 @@
 ## Installation tips
 
-If you are not (yet) an expert in Python installations (conda vs pip, mananging environements, etc.),
-here we propose a simple recipe to install `spikeinterface` and several sorters inside an anaconda
-environment for windows/mac users.
+If you are not (yet) an expert in Python installations, a main difficulty is choosing the installation procedure.
+The main ideas you need to know before starting:
+ * python itself can be distributed and installed many many ways.
+ * python itself do not contain so many features for scientifique computing you need to install "packages".
+   Numpy, scipy, matplotlib, spikeinterface, ... are python packages that have a complicated dependency graph between then. "uv"
+ * installing package  can be distributed and installed several ways (pip, conda, uv, mamba, ...)
+ * installing many packages at once is challenging (because of the depenency graph) so you need to do it in an "isolated environement"
+   to not destroy any previous installation. You need to see an "environement" as a sub installtion in dedicated folder.
+
+Choosing the installator + a environement manager + a package installer is a nightmare for beginners.
+The main options are:
+  * use "anaconda" that do everything. The most popular but bad idea because : ultra slow and agressive licensing (not free anymore)
+  * use python from the system or python.org + venv + pip : good idea for linux users.
+  * use "uv" : a new, fast and simple. We recommand this for beginners on evry os.
+
+Here we propose a steps by step recipe for beginers based on "uv".
+We used to propose here a solution based on anaconda. It is kept here for a while but we do not recommand it anymore.
+
 
 This environment will install:
  * spikeinterface full option
  * spikeinterface-gui
- * phy
- * tridesclous
+ * kilosort4
 
 Kilosort, Ironclust and HDSort are MATLAB based and need to be installed from source.
 
 ### Quick installation
 
-Steps:
+1. On macOS and Linux. Open a terminal and do
+   `$ curl -LsSf https://astral.sh/uv/install.sh | sh`
+1. On windows. Open a powershell and do
```

Comment:
Hey Sam we recommend users use Command Prompt since our shell scripts for sorters only work in command prompt. It might be better if they have a command prompt version of instructions.

Reply:
could you test if the windows command also works in CMD ?
uv docs recommand powershell (I tested and it works)